src/combine.js

```diff
@@ -4,7 +4,6 @@ import {Provider} from './providers'
  * @name combine
  * @description Combine the reducers
  */  
-export default ( ...reducers ) => {
-  let len =  Provider._reducers.length
-  reducers.forEach( reducer => { Provider._reducers[len + 1] = reducer; len++ })
-}
\ No newline at end of file
+export default (...reducers) => {
+  Provider._reducers.push(...reducers);
```

Comment:
This also has a behavior change.  The old code left a gap in the array:

Assume an empty `_reducers` array (`len = 0`).  Then the following code would skip that index during the assignment of new items:

```javascript
reducers.forEach( reducer => { Provider._reducers[ len + 1 ] = reducer; len++ }
...
reducers.forEach( reducer => { Provider._reducers[ 0 + 1 ] = reducer; len++ }
...
reducers.forEach( reducer => { Provider._reducers[ 1 ] = reducer; len++ } // No 0 entry.
```